app/admin/ministry.rb

```diff
@@ -34,7 +33,7 @@
     f.semantic_errors
     f.inputs do
       f.input :name
-      f.input :description, as: :ckeditor
```

Comment:
I did a quick search and don't see the CKeditor in use anywhere else in this project please confirm and remove the 3rd party lib if so.